records/2023-11-24.system-status.md

```diff
@@ -83,18 +83,12 @@ The API Lambda will interact with 2 different systems:
 We will write all the above results into a singular .json and commit it to github.
```

Comment:
does this need to be changed? as we are now writing the .json to an s3 bucket?

Reply:
line #75 as well

Reply:
We could still write/read the json file from github.. I hadn't intended to change that, but perhaps we should for resiliency sake?

Reply:
I will update this to use S3 as well.